pmd-core/src/main/java/net/sourceforge/pmd/ant/Formatter.java

```diff
@@ -57,12 +61,26 @@ public Renderer getRenderer() {
     }
 
     public void start(String baseDir) {
+
+        Properties properties = createProperties();
+        
+        Charset charset;
+        {
+            String s = (String) properties.get("encoding");
+            if (null == s) {
+                charset = StandardCharsets.UTF_8;
+                properties.put("encoding", charset.name());
+            } else {
+                charset = Charset.forName(s);
+            }
+        }
+
         try {
             if (toConsole) {
-                writer = new BufferedWriter(new OutputStreamWriter(System.out));
+                writer = new BufferedWriter(new OutputStreamWriter(System.out, charset));
```

Comment:
> Should the console should always use the platform encoding?

On every platform but Windows I would say yes. On Windows it's trickier because the platform's encoding is different from the console's encoding. For instance on my Windows, the platform's encoding is Cp1252 (`file.encoding` property) and the console's encoding is IBM850 (`sun.stderr.encoding` property).
So, if you want to view directly the output in the console, use IBM850, but if you redirect the output to a file, you may want to use Cp1252 or UTF-8 for instance.

So, again, it boils down to *always set the encoding parameter*, depending on your needs.